modules/tinymce/src/themes/silver/test/ts/browser/editor/autocomplete/AutocompleteTest.ts

```diff
@@ -766,4 +766,12 @@ describe('browser.tinymce.themes.silver.editor.autocomplete.AutocompleteTest', (
     },
     assertion: (editor) => TinyAssertions.assertContent(editor, '<p>multi-bB</p>')
   }));
+
+  it('TINY-11273: Autocomplete should be triggered for content with formatting applied', async () => {
```

Comment:
Can we add another formatting test like `bold` to make sure the fix works for spans and other inline tags like `strong`

Reply:
Added ✅